README.md

```diff
@@ -1,96 +1,179 @@
+
+## Standard Deployment
+```
+pcs -s standard --runtime java
+```
+
+## Basic Deployment
+```
+pcs --runtime java
+```
+
+## Local Deployment
+To set up a Remote Monitoring Solution locally see
+[Running all PCS microservices locally](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#running-all-pcs-microservices-locally)
+
+Common Scenarios
+================
+
+## All done? Connect a device!
+By default the solution uses simulated devices. You can start adding your
+own devices with the instructions here [Connect a physical device](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#connect-a-physical-device)
+
+Standard vs. Basic Deployments
+==============================
+### Standard
+The standard deployment is a production-ready deployment a developer can
+customize and extend to meet their needs. For reliability and scale, application
+microservices are built as Docker containers and deployed using an orchestrator
+([Kubernetes](https://kubernetes.io/) is default). The orchestrator is
+responsible for deployment, scaling, and management of the application.
+
+Creating a Standard solution will result in the following Azure services being
+provisioned into your Azure subscription at cost: 
+* 1 Azure Active Directory application
+* 1 Azure Container Service 
+* 3 Virtual Machines (Standard D1 V2 (1 core, 3.5GB memory))
+* 1 IoT Hub (S1 – Basic tier)
+* 1 Cosmos DB account (Standard)
+* 1 Storage account (Standard -GRS)
+* 1 Web Application
+
+### Basic
+Basic deployment is geared toward showcasing the solution. To reduce the cost
+of this demonstration, all of the microservices are deployed in a single
+virtual machine; this is not considered a production-ready architecture.
+ 
+Our Standard deployment option should be used when you are ready to customize
+a production-ready architecture, built for scale and extensibility.
+ 
+Creating a Basic solution will result in the following Azure services being
+provisioned into your Azure subscription at cost: 
+* 1 Azure Active Directory application
+* 1 Virtual Machine (Standard D1 V2 (1 core, 3.5GB memory))
+* 1 IoT Hub (S1 – Basic tier)
+* 1 Cosmos DB account (Standard)
+* 1 Storage account (Standard -GRS)
+* 1 Web Application
+
+> Pricing information for these services can be found
+[here](https://azure.microsoft.com/pricing). Usage amounts and billing details
+for your subscription can be found in the
+[Azure Portal](https://portal.azure.com/).
+
+Architecture Overview
+=====================
+<div align="center">
+<img src="https://user-images.githubusercontent.com/3317135/31836797-5a6656a0-b58b-11e7-9b64-f31da8bcd10e.png" width="700" height="auto"/>
+</div>
+
+[Open in draw.io][draw-io-map]
+
+## Microservices & Docker Containers
+Remote Monitoring is the first of our preconfigured solutions to leverage a
+microservices architecture available in both .NET and Java. Microservices have
```

Comment:
...microservices architecture.  The solution is available in both .NET and Java.